cmake/setup_finalize.cmake

```diff
@@ -68,6 +68,10 @@ FOREACH(_feature ${DEAL_II_FEATURES})
   ENDIF()
 ENDFOREACH()
 
+# we require C++11:
+FILTER_SYSTEM_LIBRARIES(CXX11)
+REGISTER_FEATURE(CXX11)
```

Comment:
The "feature" can be removed entirely. There are no libraries or headers or whatever attached to it.